client/src/containers/blog-post/BlogPostDetailsFragment/BlogPostDetailsFragment.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Col, Row, Tag, Typography } from 'antd';
+import useWindowSize from '../../../hooks/useWindowSize';
+
+import config from '../../../config';
+
+const { Title, Paragraph } = Typography;
+const { POSTS_URL } = config;
+
+const BlogPostDetailsFragment = () => {
+  const { blogPostId } = useParams();
+  const windowSize = useWindowSize();
+  const [state, setState] = React.useState({
+    postDetails: null,
+    requestError: false,
+  });
+  React.useEffect(() => {
+    fetch(`${POSTS_URL}/${blogPostId}`)
+      .then((res) => (res.status === 200 ? res.json() : null))
+      .then((postDetails) => {
+        setState((prevState) => ({
+          ...prevState,
+          postDetails,
+          requestError: postDetails === null,
+        }));
+      })
+      .catch(() => {
+        setState((prevState) => ({
+          ...prevState,
+          postDetails: null,
+          requestError: true,
+        }));
+      });
+  }, []);
+
+  if (state.postDetails === null) {
+    return state.requestError ? <p>Nu a fost găsit</p> : <p />;
+  }
+
+  return (
+    <Row
+      type="flex"
+      justify="center"
+      align="top"
+      style={{ marginTop: '2rem', marginBottom: '2rem' }}
+    >
+      <Col span={16}>
+        <Title level={3} style={{ textTransform: 'uppercase', textAlign: 'left' }}>
+          {state.postDetails.title}
+        </Title>
+        <Paragraph style={{ textAlign: 'left' }}>
+          Publicat de Autor Autor • {new Date(state.postDetails.created).toLocaleDateString()}
```

Comment:
Would you have the time to replace this with the author's name? The API now spits out `author_first_name` and `author_last_name` so the details can be added.